test/llvmpasses/fmf.jl

```diff
@@ -21,7 +21,7 @@ function dotf(a, b)
 # CHECK-NEXT:    vfmadd231pd
 # CHECK-NEXT:    vfmadd231pd
 # CHECK-NEXT:    vfmadd231pd
-        s += a[i] * b[i]
+        s = muladd(a[i], b[i], s)
```

Comment:
Note that with this PR this requires an explicit `muladd` to pass.

Reply:
Shouldn't this be a literal `fma`? `muladd` still permits the *option` to not fuse, even if in this context it almost always will.